wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```diff
@@ -154,6 +154,7 @@ public WikiServiceImpl(ConfigurationManager configManager,
 
   @Override
   public void start() {
+    initTemplates();

```

Comment:
If this method is only called at startup, how the templates will be created for the wikis created then (for example when a new group or a new user access to its wiki) ?